source/change-streams/tests/change-streams.yml

```diff
@@ -297,3 +297,163 @@ tests:
           fullDocument:
             z:
               $numberInt: "3"
+  -
+    description: Test insert, update, replace, and delete event types
+    minServerVersion: "3.6.0"
+    target: collection
+    topology:
+      - replicaset
+    changeStreamPipeline: []
+    changeStreamOptions: {}
+    operations:
+      -
+        database: *database_name
+        collection: *collection_name
+        name: insertOne
+        arguments:
+          document:
+            x: 1
+      -
+        database: *database_name
+        collection: *collection_name
+        name: updateOne
+        arguments:
+          filter:
+            x: 1
+          update:
+            $set:
+              x: 2
+      -
+        database: *database_name
+        collection: *collection_name
+        name: replaceOne
+        arguments:
+          filter:
+            x: 2
+          replacement:
+            x: 3
+      -
+        database: *database_name
+        collection: *collection_name
+        name: deleteOne
+        arguments:
+          filter:
+            x: 3
+    expectations:
+      -
+        command_started_event:
+          command:
+            aggregate: *collection_name
+            cursor: {}
+            pipeline:
+              -
+                $changeStream:
+                  fullDocument: default
+          command_name: aggregate
+          database_name: *database_name
+    result:
+      success:
+        -
+          operationType: insert
+          ns:
+            db: *database_name
+            coll: *collection_name
+          fullDocument:
+            x:
+              $numberInt: "1"
```

Comment:
I'm not familiar with the other change stream tests, but is the `_id` field purposefully omitted here? Would it make sense to use numeric `_id` values in the example data (as is done in most CRUD tests IIRC) so that we can assert it as well?

Reply:
Most of the other change stream tests don't include `_id`. I could add it here, but what benefit would asserting it provide over what we currently have?

Reply:
I may be misunderstanding the test runner behavior, but with `_id` omitted here, is the test runner supposed to completely ignore the field when evaluating the assertion? I assume the response from the server _is_ going to include `_id` and that the spec runner would assert that `fullDocument` is equal to what we have here.

Reply:
According to the README, "Assertion passes if ``expected`` is a subset of ``actual``" so the spec runner does not need to assert that the `_id` is correct, although the server response will include the `_id`.